### PR TITLE
WIP: add search and sorting

### LINKED ISSUE #10: UI: Filter by column values
aka: "all PRs assigned to foo" or "all PRs that are failed" or "all mergeable PRs", etc.

This may also be able to be accomplished by being able to sort each column

### PR DESCRIPTION
**Pull request by Gankro**
This builds ontop of #31 to implement search and sorting of the data by including the DataTable library (and its jquery dependency). For now I'm pulling them in over their respective CDNs for simplicity, but we can of course download them and include them as part of homu. Your call.

UI untouched:
screenshot from 2015-01-19 16 38 36

Searching for collections-related PRs:
screenshot from 2015-01-19 16 39 37

Sorting by assignee:
screenshot from 2015-01-19 16 40 44

Searching for niko-related PRs, sorted by assignee:
screenshot from 2015-01-19 16 40 55

Fixes #10 #9

### CODE REVIEW

**Comment by Gankro** [comment]
Rebased. Are there any outstanding issues?

**Comment by barosl** [comment]
Oh, nope. I was thinking this PR was still WIP. Is this ready to be merged?

File: homu/html/queue.html [hunk 1]
```diff
@@ -21,6 +22,10 @@
 
             .yes { color: green; }
             .no { color: red; }
+            .unknown { color: yellow; }
```
Comment (barosl) [inline homu/html/queue.html]: This seems to be the legacy from the old days!

File: homu/html/queue.html [hunk 2]
```diff
@@ -3,6 +3,7 @@
     <head>
         <meta charset="utf-8">
         <title>Homu queue - {{repo_name}}</title>
+        <link href="//cdn.datatables.net/1.10.4/css/jquery.dataTables.min.css">
```
Comment (barosl) [inline homu/html/queue.html]: I know including a stylesheet in `<link>` is recommended in the HTML standard, but wouldn't it slow down the initial load time if the CDN is down? Or, maybe invalidating the DOM due to the late CSS insertion would be more critical? What's your opinion on this?

Comment (barosl) [inline homu/html/queue.html]: Inspecting the HTTP response from their server, it seems that they do provide the `Expires` header, so there wouldn't be much trouble... But I just want to make sure of this.

File: homu/html/queue.html [hunk 3]
```diff
@@ -100,6 +108,14 @@
...
             handle_auto_reload();
+
+            $(document).ready(function(){
+                $('#data').DataTable({
+                    paging: false,
+                    order: [],
+                });
+            });
         </script>
+
```
Comment (barosl) [inline homu/html/queue.html]: There's an extra empty line in here!

File: homu/html/queue.html (hunk 1 above)
Comment (Gankro) [inline homu/html/queue.html]: Ah sorry, rebase error.

File: homu/html/queue.html (hunk 2 above)
Comment (Gankro) [inline homu/html/queue.html]: On a lark, I just completely removed it, and everything seems to be working fine?

**Comment by Gankro** [comment]
issues addressed.

File: homu/html/queue.html (hunk 2 above)
Comment (barosl) [inline homu/html/queue.html]: Huh, does it? Indeed, styles will not be that significant considering the current usage in the code. Maybe we can remove it entirely? :trollface:

**Pull request merged**